docs/reference-manual/native-image/DebugInfo.md

```diff
@@ -4,82 +4,69 @@ toc_group: native-image
 link_title: Debug Info Feature
 permalink: /reference-manual/native-image/DebugInfo/
 ---
+
 # Debug Info Feature
 
-To add debug info to a generated native image, add the flag
-`-H:GenerateDebugInfo=<N>` to the native image command line (where `N` is
-a positive integer value -- the default value `0` means generate no
-debug info). For example,
+To add debug information to a generated native image, provide the `-g -O0` option to the `native-image` builder:
```

Comment:
I agree, we should steer user towards using the API options instead of raw HostedOptions.

But we should not mention `-O0` here. While it does make the debugging experience better it's not correct to state it has anything to do with debuginfo generation.

Reply:
Understood, thanks Paul